### DISCUSSION TITLE
ESP32 Solo 1?

### DISCUSSION THREAD

**Discussion by turambar517** [pschatzmann/ESP32-A2DP#521] [Q&A] [answered]
Hi there, any reason to think that the solo 1 wouldn't work with this library? It supports BT classic and the only difference seems to be processing with one core instead of two... https://www.espressif.com/sites/default/files/documentation/esp32-solo-1_datasheet_en.pdf

**Comment by pschatzmann** [comment, marked as answer]
I would not know: I never heard of this.
I would be worried that there is only one core...

Let me know about your results, if you manage to test this.
